### PR TITLE
bugfix-session

### PR DESCRIPTION
**Pull request by jampulanaveen**

### CODE REVIEW

File: razorpay-quick-payments.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Plugin Name: Razorpay Quick Payments
  * Plugin URI: https://github.com/razorpay/razorpay-quick-payments
  * Description: Quick Payments for Wordpress, by Razorpay.
- * Version: 1.2.2
+ * Version: 1.2.1
```
Comment (ChetanGN) [inline razorpay-quick-payments.php:7]: @jampulanaveen Upgrade the version to 1.2.3

File: razorpay-quick-payments.php [hunk 2]
```diff
@@ -14,13 +14,15 @@
 use Razorpay\Api\Api;
 
 require_once __DIR__.'/includes/razorpay-settings.php';
-
 session_start();
 
+
```
Comment (ChetanGN) [inline razorpay-quick-payments.php:19]: @jampulanaveen Remove unnecessary space.

File: razorpay-quick-payments.php [hunk 3]
```diff
@@ -14,13 +14,15 @@
...
 add_action('plugins_loaded', 'wordpressRazorpayInit', 0); // not sure if this is the right hook
 
 function wordpressRazorpayInit()
 {
+    wp_enqueue_script('jquery');
```
Comment (ChetanGN) [inline razorpay-quick-payments.php:24]: @jampulanaveen Why it's called twise?

File: razorpay-quick-payments.php (hunk 1 above)
Comment (jampulanaveen) [reply razorpay-quick-payments.php:7]: ok

File: razorpay-quick-payments.php (hunk 3 above)
Comment (jampulanaveen) [reply razorpay-quick-payments.php:24]: will remove it

**Review by ChetanGN — APPROVED** [review]
LGTM.

**Pull request merged**